runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -388,15 +387,18 @@ public void draw(MainBufferProvider mainBufferProvider, Graphics graphics, int x
 		final Image finalImage;
 		if (client.isStretchedEnabled())
 		{
-			GraphicsConfiguration gc = clientUi.getGraphicsConfiguration();
+			GraphicsConfiguration gc = ClientUI.getGraphicsDevice().getDefaultConfiguration();
 			Dimension stretchedDimensions = client.getStretchedDimensions();
 
 			if (lastStretchedDimensions == null || !lastStretchedDimensions.equals(stretchedDimensions)
-				|| (stretchedImage != null && stretchedImage.validate(gc) == VolatileImage.IMAGE_INCOMPATIBLE))
+				|| (stretchedImage != null && stretchedImage.validate(gc) != VolatileImage.IMAGE_OK))
 			{
-				/*
-					Reuse the resulting image instance to avoid creating an extreme amount of objects
-				 */
+				// Only reuse image if it is unsullied else reconstruct new
```

Comment:
how is the image reused here if the line below this creates a new image? what is the purpose of flushing the image if that is the case?